src/klein/test/test_message.py

```diff
@@ -18,18 +19,18 @@
 __all__ = ()
 
 
-class FrozenHTTPMessageTestsMixIn(object):
+class FrozenHTTPMessageTestsMixIn(ABC):
```

Comment:
Use an ABC and mark methods abstract instead of raising when called; this catches code errors immediately and removes untested code.